content/routing.md

```diff
@@ -322,13 +329,15 @@ If some data in your app has been moved, you probably want to redirect people to
 
 As you maintain and develop new features for your app, you might discover that you need to change your URL structure. If there are already lots of links and bookmarks to your app floating around in the wild, it might be a good idea to redirect the old URLs for backwards compatibility.
 
+// XXX: This can be both done in the server side and client. In FR 3.0, we'll make this isomorphic.
```

Comment:
I feel like it could just make things confusing? If people want to learn about the future of Flow Router, they can learn about it on the Flow Router website. When the new version is released we can update the guide. What do you think?